### PR TITLE
Add dripline post about mai's piece

### LINKED ISSUE #462: Publish Mai's piece about "Emerging practices on the decentralized cooperative web" on Dripline
We should publish [this in July](https://blog.opencollective.com/emergent-practices-from-the-decentralized-co-operative-web/)

### PR DESCRIPTION
**Pull request by uditvira**
closes hyphacoop/organizing#462

one question I had was: should we say that this was first published on oc blog on x date?

### CODE REVIEW

**Review by benhylau — APPROVED** [review]

**Comment by benhylau** [comment]
I think we could add:

>_This article is also published on [Open Collective's blog](https://blog.opencollective.com/emergent-practices-from-the-decentralized-co-operative-web/)._

**Comment by dcwalk** [comment]
Yes I think we need language like "This is a guest post originally published at X on X."

File: _posts/2021-09-07-emergent-practices.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+title: 'Emergent Practices from the Decentralized Co-operative Web'
+author: Mai Ishikawa Sutton
+date: 2021-09-02
+acknowledgement: By Mai Ishikawa Sutton, Co-founder and Lead Editor of [COMPOST Magazine][compost-mag]
+excerpt: 'Building tools and practices that enable organizations to work more transparently, equitably, and collaboratively.'
+---
+_{{page.acknowledgement}}_
```
[the remark this answers is not in the document]
Comment (dcwalk) [reply _posts/2021-09-07-emergent-practices.md]: Or for now, if we need to add something here, would say:

```suggestion
_By {{ page.author }}, {{ page.acknowledgement }}_    
```

File: _posts/2021-09-07-emergent-practices.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
+---
+title: 'Emergent Practices from the Decentralized Co-operative Web'
+author: Mai Ishikawa Sutton
+date: 2021-09-02
+acknowledgement: By Mai Ishikawa Sutton, Co-founder and Lead Editor of [COMPOST Magazine][compost-mag]
```
Comment (dcwalk) [inline _posts/2021-09-07-emergent-practices.md]:
```suggestion
acknowledgement: Mai is the Co-founder and Lead Editor of [COMPOST Magazine][compost-mag]. This is a  guest post that was originally published on the [Open Collective Blog][open-collective-post] on May 27, 2021.
```

File: _posts/2021-09-07-emergent-practices.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  [sourcecred]: https://sourcecred.io/
+  [radicle]: https://radicle.xyz/
+  [fundoss]: https://fundoss.org/
+  [disco]: https://disco.coop/
+  [disco-elements]: https://elements.disco.coop/
+  [hypha]: https://hypha.coop/
+  [hypha-handbook]: https://handbook.hypha.coop/
+  [emergent-strategy]: https://www.akpress.org/emergentstrategy.html
+  [dweb-principles]: https://getdweb.net/principles/
+
```
Comment (dcwalk) [inline _posts/2021-09-07-emergent-practices.md:85]:
```suggestion
[open-collective-post]: https://blog.opencollective.com/emergent-practices-from-the-decentralized-co-operative-web/
```

File: _posts/2021-09-07-emergent-practices.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+The projects highlighted here are experimenting with this type of radical, pragmatic transparency. They’re the kinds of tools that will help people break out of the dominant economic logic that continues to perpetuate social inequities. I’m excited to see how these tools for economic solidarity will grow and lead to even more experimentation. Eventually, they may help us establish radically better practices on creating organizations that are truly, organizationally, decentralized and interdependent.
+
+*** 
+
+## Additional Reading
+* [COMPOST Magazine][compost-mag]
+* [Hypha Organizational Handbook][hypha-handbook]
+* [DisCO Elements][disco-elements]
+* [Emergent Strategies by adrienne maree brown][emergent-strategy]
+* [Decentralized Web Principles][dweb-principles]
```
Comment (dcwalk) [inline _posts/2021-09-07-emergent-practices.md:57]: Something wrong with how this renders, I see a paragraph here: https://staging.hypha.coop/dripline/emergent-practices/ can you re-flow this list to see if it resolves?

**Review by dcwalk — CHANGES_REQUESTED** [review]
Left feedback inline, I thought the layout changes we happeing prior to this PR? did I miss that?

File: _posts/2021-09-07-emergent-practices.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+## Addressing the accessibility of tech tools
+Across the board, there was an agreement that many of the tools we build and use remain inaccessible to many. Scott acknowledged how there’s still a lot of work to be done in the cryptocurrency ecosystem, as well as in open source software where major technical contributors are prioritized over individual members of the community. We then discussed how with greater cooperation, tooling, and patterns of working together, that the tech side of projects could just be another specialization, as a role that could be designated to someone. This was part of a greater recognition that we need more diversity within projects to specialize in different aspects of an organization. Tools like Open Collective and organizations like Hypha Co-operative help remove the financial, legal, and administrative barriers for those trying to build their project or address frontline needs, by enabling them to outsource this work.
+
+~
```
Comment (dcwalk) [inline _posts/2021-09-07-emergent-practices.md]: This as a text mark doesn't really flow with text
```suggestion

```

**Comment by uditvira** [comment]
made updates, can @dcwalk and @ASoTNetworks review?

File: _posts/2021-09-07-emergent-practices.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (uditvira) [reply _posts/2021-09-07-emergent-practices.md]: see latest

File: _posts/2021-09-07-emergent-practices.md (hunk 3 above)
Comment (uditvira) [reply _posts/2021-09-07-emergent-practices.md:85]: see latest

File: _posts/2021-09-07-emergent-practices.md (hunk 5 above)
Comment (uditvira) [reply _posts/2021-09-07-emergent-practices.md]: see latest

File: _posts/2021-09-07-emergent-practices.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
+---
+title: 'Emergent Practices from the Decentralized Co-operative Web'
```
Comment (dcwalk) [inline _posts/2021-09-07-emergent-practices.md]:
```suggestion
title: 'Emergent practices from the decentralized co-operative web'
```

File: _posts/2021-09-07-emergent-practices.md (hunk 4 above)
Comment (uditvira) [reply _posts/2021-09-07-emergent-practices.md:57]: see latest

**Pull request merged**